README.rst

```diff
@@ -108,7 +108,8 @@ You can instantiate the IonQ devices for PennyLane as follows:
 
     import pennylane as qml
     dev1 = qml.device('ionq.simulator', wires=2, shots=1000)
-    dev2 = qml.device('ionq.qpu', wires=2, shots=1000)
+    dev2 = qml.device('ionq.qpu.harmony', wires=2, shots=1000)
+    dev3 = qml.device('ionq.qpu', backend='aria-1', wires=2, shots=1000)
```

Comment:
The documentation in the readme is updated here (pending modification to remove the device-specific subclassing), but the plugin documentation needs a similar update. The `docs/devices.rst` file needs to be updated to show the new `backend` functionality, and to refer users to IonQ’s website to find out what backends are available.